etc/greenboot/green.d/50_greenboot_motd.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
 set -euo pipefail
 
-ln -snf /etc/greenboot/motd/greenboot.motd /run/greenboot/motd
+if [ ! -d /etc/motd.d ]; then
+    mkdir /etc/motd.d
+fi
+
+greenboot_logs=`journalctl -u greenboot.service -p 0..5 -b -0 -o cat`
+printf '%s\n' "$greenboot_logs" > /etc/greenboot/motd
```

Comment:
are we writing to `/etc/greenboot/motd` on every boot? seems like we should be writing to somehwere in /run/ instead?

Reply:
with @rfairley's changes, we could probably do this without symlinking and instead writing directly to `/run/motd.d/greenboot`

Reply:
yes - hopefully when the changes land the symlink can be avoided - hope to get something up for review soon.

I do something similar in the `motd` [package I had been working on](https://github.com/rfairley/fedora-coreos-login-messages) - [writes a new motd](https://github.com/rfairley/fedora-coreos-login-messages/blob/master/usr/lib/coreos/motdgen#L66) to `/run/${%name}.motd` with it symlinked from `/etc/motd.d`. With the PAM changes `/run/motd.d` should become a "canonical" directory to do this for any package.

Reply:
I guess as of now, writing to `/etc` is the best option,  as symlinks to `/run` alone won't work due to the modifications needed in the selinux policy for PAM to read the file in `/run`. But the changes in PAM (which will also bring the modified selinux policy for PAM ot read `/run/motd.d`) should avoid the need in the future to make a symlink to run.

I think the writing to `/etc` is good to merge for now - to not block on the PAM changes

Reply:
+1